app/browser/api/ledger.js

```diff
@@ -90,12 +90,17 @@ let runTimeoutId
 let promotionTimeoutId
 let togglePromotionTimeoutId
 let verifiedTimeoutId = false
+let verifiedPTimeoutId = false
```

Comment:
Definitely a good call.

I'm going to go through and clean up the conventions, also since this endpoint includes `exclude` info I may remove the verified prefix common throughout this PR